provider/runbook_resource.go

```diff
@@ -163,6 +173,10 @@ func createResourceFireHydrantRunbook(ctx context.Context, d *schema.ResourceDat
 		Type:        d.Get("type").(string),
 	}
 
+	if ownerID, ok := d.GetOk("owner_id"); ok && ownerID.(string) != "" {
+		createRequest.Owner = &firehydrant.RunbookTeam{ID: ownerID.(string)}
+	}
+
```

Comment:
nit: would you mind moving this down under the "Process any optional attributes" comment like you did in the update method?